web/src/util/services/templates.ts

```diff
@@ -84,6 +84,6 @@ function manifestFromInstaller(i: Installer, kurlURL: string, replicatedAppURL:
     KURL_UTIL_IMAGE: kurlUtilImage,
     KURL_BIN_UTILS_FILE: kurlBinUtils,
     STEP_VERSIONS: `(${Installer.latestMinors().join(" ")})`,
-    INSTALLER_YAML: i.toYAML(),
+    INSTALLER_YAML: i.toYAML().replace(/\"/g, "\\\""),
```

Comment:
doesn't this have the possibility of breaking valid yaml?

Reply:
Or is there a "bash string escape" function we could be using instead?

Reply:
also - shouldn't we be escaping double-quotes? (Like, `abc: "abc"` should become `abc: \"abc\"`)

if we want this to be a fully-general solution, that is

Reply:
This was for double quotes, are you thinking we should do single quotes as well?

Reply:
Discussed elsewhere - we should make a "bashEscape" function that replaces `\` with `\\` and then `"` with `\"`